src/hotspot/share/gc/shenandoah/shenandoahOldGC.cpp

```diff
@@ -44,6 +81,12 @@ bool ShenandoahOldGC::collect(GCCause::Cause cause) {
   // Complete marking under STW
   vmop_entry_final_mark();
 
+  ShenandoahOldHeuristics* old_heuristics = heap->old_heuristics();
+
+  entry_coalesce_and_fill();
+
+  old_heuristics->start_old_evacuations();
```

Comment:
Could you encapsulate in entry_old_evacuations() with proper timing/logging?  it really helps future debugging the crashes with hs_err file.

Reply:
Thanks.  I've created new function entry_old_evacuations(), which calls entry_coalesce_and_fill() followed by old_heuristics->start_old_evacuations().  There is already timing/logging in entry_coalesce_and_fill().  old_heuristics->start_old_evacuations() just "flips a switch" so that subsequent evacuations will fold in old-gen collection sets so I don't think we need any timing there.

This will show up in next commit.